addon/styles/addon.less

```diff
@@ -2,8 +2,10 @@
 @import (optional) "./variables.less";
 //"bg" bundles only
 @import (optional) "node_modules/bg-vi/src/styles/custom-bg/variables.less";
+@import (optional) "node_modules/bg-vi/src/styles/bgcss/bgcss-variables.less";
 @import (optional) "node_modules/bg-vi/src/styles/brands/sainsburys/variables.less";
 @import (optional) "node_modules/bg-vi/src/styles/brands/britishgas/variables.less";
+@import (optional) "node_modules/ember-commons/addon/styles/z-indexes.less";
```

Comment:
Ember-coomns and bg-vi are private packages, doesnt the lack of those packages mess up public users?

Reply:
@gczene no it's optional imports so will just fail silently if files are not there.  Didn't find another way for it to work in all cases, with the less variables needed(whether BG app or not). Open to suggestions though.

Reply:
So BG apps can use bg-vi and ember-commons CSS, through tree-shaking, "BG" bundle, which removes all the files needed by public users but that we want to consume from bg-vi / commons instead (CSS, services, bg-button etc.)